examples/pytorch/semantic-segmentation/run_semantic_segmentation.py

```diff
@@ -411,7 +414,7 @@ def compute_metrics(eval_pred):
     val_transforms = Compose(
         [
             ReduceLabels() if data_args.reduce_labels else Identity(),
-            Resize(size=(feature_extractor.size, feature_extractor.size)),
+            Resize(size=size),
```

Comment:
Here it's possible that images will have a different size, as torchvision's Resize with an integer size will only resize the shorter edge of the image, keeping the aspect ratio. This means that batching images won't be possible.

Reply:
In other words, one needs to always define (height, width) here, otherwise it will fail

Reply:
I made it such that from the size dict, size is either:
* `(feature_extractor.size["height"], feature_extractor.size["width"])`
* `(feature_extractor.size["shortest_edge"], feature_extractor.size["shortest_edge"])`